src/assets/locales/fr.json

```diff
@@ -337,23 +382,28 @@
   "preferences-screen.material-quality-setting.medium": "Medium",
   "preferences-screen.preference.allow-multiple-hubs-instances": "Désactiver la sortie automatique quand de multiples instances de hubs sont ouvertes",
   "preferences-screen.preference.animate-waypoint-transitions": "Animer les transitions entre les waypoints",
+  "preferences-screen.preference.audio-clipping-threshold": "Seuil pour l'audio clipping",
```

Comment:
"preferences-screen.preference.audio-clipping-threshold": "Audio Clipping Threshold",

Reply:
```suggestion
  "preferences-screen.preference.audio-clipping-threshold": "Seuil de coupure audio",
```

Reply:
Not sure about what it does ? @vincentfretin is it possible to explain ?

Reply:
ok for "coupure audio"